docs/api/README.md

```diff
@@ -4,226 +4,71 @@
 
 ```js
 {
-  // Default styles
-  margin: 0,
-
-  // Default browser states
-  states: [
-    {
-      hover: {
-        background: '#000'
-      }
+var styles = {
+  base: {
+    backgroundColor: "#0074d9",
+    border: 0,
+    borderRadius: "0.3em",
+    color: "#fff",
+    cursor: "pointer",
+    fontSize: 16,
+    outline: "none",
+    padding: "0.4em 1em",
+
+    // Media queries must start with @media, and follow the same syntax as CSS
+    '@media (min-width: 992px)': {
+      padding: "0.6em 1.2em"
     },
-    {
-      focus: {
-        background: '#333'
-      }
-    },
-    {
-      active: {
-        background: '#666'
-      }
-    }
-  ],
 
-  // Default media queries
-  mediaQueries: [
-    {
-      small: {
-        padding: 10
-      }
+    '@media (min-width: 1200px)': {
```

Comment:
Can we add an example of `:hover` *inside* a media query to illustrate how that works?